port/unix/omrsysinfo.c

```diff
@@ -1564,14 +1583,25 @@ omrsysinfo_get_s390_description(struct OMRPortLibrary *portLibrary, OMRProcessor
 	/* zEC12 facility and processor detection */
 
 	/* TE/TX hardware support */
-	if (omrsysinfo_test_stfle(portLibrary, 50) && omrsysinfo_test_stfle(portLibrary, 73)) {
+	if (omrsysinfo_test_stfle(portLibrary, 73)) {
```

Comment:
Should the `73` (and `50` below on L1597) use the new OMR_FEATURE_* macros you added in omrport.h?

```
/* STFLE bit 50 - Constrained transactional-execution facility */
#define OMR_FEATURE_S390_CONSTRAINED_TRANSACTIONAL_EXECUTION_FACILITY 50
/* STFLE bit 73 - Transactional-execution facility */
#define OMR_FEATURE_S390_TRANSACTIONAL_EXECUTION_FACILITY 73
```